content/4_2_rethink_db.tex

```diff
@@ -0,0 +1,412 @@
+\chapter{Document Oriented}
```

Comment:
We already have a chapter for document oriented database, once `couchbase` is merged.